ui/src/components/form/context.js

```diff
@@ -19,7 +19,9 @@ export const FormContextProvider = ({ data: initData, ...props }) => {
     [setData]
   );
 
-  const rootHandler = useCallback(() => new StackableFunction([], handleChanges), [
+  // TODO: The eslint rule below is disabled. Consider refactoring to conform to the rule.
```

Comment:
@davidheryanto after checking the code, my understanding is that we want the `new StackableFunction` object to be:
* The argument to the `useOnChangeHandler` call in the line below ([ref](https://github.com/gojek/turing/blob/main/ui/src/components/form/context.js#L26))
* Recomputed (i.e., recreated) only when `handleChanges` is updated (so that the `useEffect` defined inside the `useOnChangeHandler` hook will not be continuously triggered)

So, instead of `useCallback` which returns a callback function, I believe `useMemo` should be used:
```
import React, { useCallback, useMemo, useState } from "react";
...
const rootHandler = useMemo(() => new StackableFunction([], handleChanges), [
    handleChanges
  ]);
```

This passes the ES lint checks and fixes the bug as well. @romanwozniak wdyt?

Reply:
looks legit :)